pom.xml

```diff
@@ -1498,6 +1498,28 @@
                 <exclude>**/JavadocPropertiesGenerator.class</exclude>
                 <!-- generated classes, unfortunately use problematic api -->
                 <exclude>**/JavadocParser.class</exclude>
+                <!-- reason at https://github.com/checkstyle/checkstyle/issues/7759 -->
+                <exclude>**/RegexpCheck.class</exclude>
+                <!-- no code in file, no AST to log -->
+                <exclude>**/NoCodeInFileCheck.class</exclude>
+                <!-- until https://github.com/checkstyle/checkstyle/issues/5770 -->
+                <exclude>**/AbstractJavadocCheck.class</exclude>
+                <exclude>**/AtclauseOrderCheck.class</exclude>
+                <exclude>**/JavadocBlockTagLocationCheck.class</exclude>
+                <exclude>**/JavadocMethodCheck.class</exclude>
+                <exclude>**/JavadocMissingLeadingAsteriskCheck.class</exclude>
+                <exclude>**/JavadocMissingWhitespaceAfterAsteriskCheck.class</exclude>
+                <exclude>**/JavadocParagraphCheck.class</exclude>
+                <exclude>**/JavadocStyleCheck.class</exclude>
+                <exclude>**/JavadocTagContinuationIndentationCheck.class</exclude>
+                <exclude>**/JavadocTypeCheck.class</exclude>
+                <exclude>**/MissingDeprecatedCheck.class</exclude>
+                <exclude>**/NonEmptyAtclauseDescriptionCheck.class</exclude>
+                <exclude>**/RequireEmptyLineBeforeBlockTagGroupCheck.class</exclude>
```

Comment:
It might be the only Check need to fix, all others are javadoc and we do no support suppression by ast for them.
So reporting by ast might be too early.
If you agree, please group all javadoc together and keep only java Check until this issue.

Reply:
This is also a Javadoc check, all checks in this group are. I just copy pasted list from https://checkstyle.sourceforge.io/config_filters.html#SuppressionXpathFilter :)

Reply:
I believe the check being discussed here is `JavadocTypeCheck` as it is the only one of the 4 that extends `AbstractCheck`, but even this seems like an outdated javadoc regexp check.

Reply:
Yes, it is javadoc check
https://github.com/checkstyle/checkstyle/blob/6f000bfbd10948a729befe4ec5cace79f30f2f5b/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/RequireEmptyLineBeforeBlockTagGroupCheck.java#L103

Ok, all of them are same group